hypertrace-view-generator/hypertrace-view-generator/src/test/resources/configs/view-gen-log-event/application.conf

```diff
@@ -0,0 +1,39 @@
+service.name = view-gen-raw-service
```

Comment:
Do you need this file? Is it used?

Reply:
yes, for integration test in Hypertrace-ingester